scripts/bind_to_driver.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Usage: ./bind_to_driver.sh <ssss:bb:dd.f> <driver>
+# Bind the GPU specified by the PCI_ID=ssss:bb:dd.f to the given driver.
+
+bind_to_driver()
```

Comment:
Why not implement this in go?

Reply:
Atleast for `unbind`, in case `unbindLock` is in use, we'd need the same process that acquires the `unbindLock` to be the one doing the driver `unbind`. I want to scope that `unbindLock` to just the task that's doing the `unbind` rather than the whole `kubeletplugin` binary. 

(FYI `unbindLock` is in play when the NVIDIA Grid-vGPU driver is used on the node)

`bind` is a script just to be consistent with `unbind`.